apps/client/src/common/components/view-params-editor/constants.ts

```diff
@@ -1,106 +1,122 @@
 import { UserFields } from 'ontime-types';
+import { TimeFormat } from 'ontime-types/src/definitions/core/TimeFormat.type';
 
 import { ParamField } from './types';
 
-export const TIME_FORMAT_OPTION: ParamField = {
+export const getTimeOption = (timeFormat: TimeFormat): ParamField => ({
   id: 'format',
   title: '12  / 24 hour timer',
   description: 'Whether to show the time in 12 or 24 hour mode. Overrides the global setting from preferences',
   type: 'option',
   values: { '12': '12 hour AM/PM', '24': '24 hour' },
-};
+  defaultValue: timeFormat,
+});
 
-export const CLOCK_OPTIONS: ParamField[] = [
-  TIME_FORMAT_OPTION,
+export const getClockOptions = (timeFormat: TimeFormat): ParamField[] => [
+  getTimeOption(timeFormat),
   {
     id: 'key',
     title: 'Key Colour',
     description: 'Background colour in hexadecimal',
     type: 'string',
+    defaultValue: '101010',
   },
   {
     id: 'text',
     title: 'Text Colour',
     description: 'Text colour in hexadecimal',
     type: 'string',
+    defaultValue: 'fffff',
   },
   {
     id: 'textbg',
     title: 'Text Background',
     description: 'Colour of text background in hexadecimal',
     type: 'string',
+    defaultValue: '101010',
```

Comment:
maybe this should default to`00000000` <- notice the last to zeros makes it transparent
this way users only have to change the Key Colour

Reply:
I dont actually understand the comment. The values are used if they exist, otherwise we use the ones defined in CSS. In this case the value of the background colour is `transparent`

since the user cannot use transparent as a key, the 00.00.00.00 suggested is the closest.
It might also be cleaner to have an empty field in the cases where it is unclear

Reply:
Fair, I'll use `00000000` then

Reply:
Resolved with https://github.com/cpvalente/ontime/commit/837d6de03de7923b490683dd872e213425c186b9